acceptance/config/beaker/jenkins/el-7-x86_64-mdca.cfg

```diff
@@ -8,6 +8,14 @@ HOSTS:
     platform: el-7-x86_64
     template: Delivery/Quality Assurance/Templates/vCloud/centos-7-x86_64
     hypervisor: vcloud
+    puppetcodedir: /etc/puppetlabs/code
+    distmoduledir: /etc/puppetlabs/code/modules
+    hieraconf:     /etc/puppetlabs/code/hiera.yaml
+    puppetconfdir: /etc/puppetlabs/puppet
+    puppetserver-confdir: /etc/puppetserver/conf.d/
```

Comment:
Dunno yet, I'll have to run it through CI and post back if it's working or not.  Since the goal is to get things green, I'm going to be following Josh Cooper's lead and approach, creating a `XXX_hacks.rb` file that has things we know to be "wrong" but are "right" in that they get CI to a known good state as quickly as possible.

FWIW, I pulled this value from https://github.com/puppetlabs/puppet/commit/70feeef3810850c99199f9e702048848c0f02bf3#diff-972ce103aac1e41c0a3dca0de9bc6a7bR13 via https://tickets.puppetlabs.com/browse/PUP-3632